test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/io/SeqInputStreamForTest.scala

```diff
@@ -14,11 +14,6 @@ package org.scalajs.testsuite.javalib.io
 
 import java.io._
 
-import org.junit.Test
-import org.junit.Assert._
-
-import org.scalajs.testsuite.utils.AssertThrows.assertThrows
```

Comment:
These changes should be in the first commit, the one that introduced `SeqInputStreamForTest`.